docs/rfcs/GPUXDialect.md

```diff
@@ -0,0 +1,233 @@
+# RFC: GPUX Dialect (Extension of upstream GPU Dialect)
+
+Core MLIR Team
+
+## Summary
+
+We propose the GPUX dialect as an extension of the upstream GPU dialect. The GPUX dialect allows us to expose stream creation/destruction ops.These ops are required by the underlying runtimes (Level zero & Sycl) for explicit stream/context/device creation.
```

Comment:
I think you can switching the order of the statements may look more natural. You can first mention some runtimes require explicit stream context and device since that is the reason for introducing GPUX dialect. And then move onto introducing new ops (stream creation/destruction) and op changes.